app/controllers/v1/sessions_controller.rb

```diff
@@ -24,15 +24,23 @@ def new
 
       StatsD.increment(STATSD_SSO_NEW_KEY, tags: ["context:#{type}"])
       url = url_service.send("#{type}_url")
+
       if type == 'slo'
         Rails.logger.info('SSO: LOGOUT', sso_logging_info)
         reset_session
+      elsif type == 'ssoe_slo'
+        Rails.logger.info('SSOe: LOGOUT', sso_logging_info)
+        reset_session
       end
       # clientId must be added at the end or the URL will be invalid for users using various "Do not track"
       # extensions with their browser.
       redirect_to params[:client_id].present? ? url + "&clientId=#{params[:client_id]}" : url
     end
 
+    def ssoe_slo_callback
```

Comment:
This is the URL we can have SSOe redirect to after logout is called. This method will allow us to have control over where the user ultimately lands after logout.

Reply:
Could you just make a change here? https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/saml/url_service.rb#L57

Reply:
I think we still need this method because it is tied to a callback url that SSOe will call after they successfully trigger logout. https://github.com/department-of-veterans-affairs/vets-api/blob/972f0d7dd46478b88316a34811b186e67336a608/config/routes.rb#L20

Right now it just calls the redirect URL in url_service and we can make changes later on to the URL either here or in the URL service if needed.